src/request-message-processor.js

```diff
@@ -84,7 +70,10 @@ export class RequestMessageProcessor {
             // Some interceptors can delay sending of XHR, so when abort is called
             // before XHR is actually sent we abort() instead send()
             this.xhr.abort();
-          } else {
+          }
+          else {
```

Comment:
It'd be great to match the existing style, and leave the `else` on the same line as the previous closing brace.